### PR TITLE
Module 1:  Contact form

### PR DESCRIPTION
**Pull request by boniface193**
### Third milestone of my portfolio website

 - Created form with HTML5 validations.
 - collaborated with coding partner
 - [Formspree service](https://formspree.io/html). was added

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -314,6 +314,58 @@ nav ul li span {
...
+.contact-sub-title {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-form {
+  margin-top: 87px;
+}
```
Comment (amejid) [inline style.css]: - [x] Your design deserves a :+1:, kindly use flexbox to align the contents of your form as it is mentioned in the project requirement.

File: index.html [hunk 2]
```diff
@@ -225,6 +225,38 @@ <h3 class="nav-sub-sub-heading weight--500 text--info mt-12">
...
+                    </aside>
+
+                    <aside>
+                        <form action="https://formspree.io/f/xknedbwy" method="post" class="contact-form">
+                            <div>
+                                <input type="text" maxlength="30" required name="fullName" placeholder="your full name"
+                                    class="weight--400">
+                            </div>
+                            <div>
+                                <input type="email" required name="email" placeholder="your name" class="weight--400">
```
Comment (amejid) [inline index.html]: - [x] I think this input is to receive the email of a user. Kindly use the correct placeholder text to give users an emphasis on what to put in that input. 😎️

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @boniface193, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Github flow :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Good UI design :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @boniface193 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

good-fine


## Highlights
All linters are passing :heavy_check_mark: 
Correct use of GitHub flow


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**